uhabits-core/src/jvmMain/java/org/isoron/uhabits/core/models/HabitList.kt

```diff
@@ -139,31 +108,28 @@ public ModelObservable getObservable()
      * @param h the habit
      * @return the index of the habit, or -1 if not in the list
      */
-    public abstract int indexOf(@NonNull Habit h);
-
-    public boolean isEmpty()
-    {
-        return size() == 0;
-    }
+    abstract fun indexOf(h: Habit): Int
+    val isEmpty: Boolean
+        get() = size() == 0
 
     /**
      * Removes the given habit from the list.
-     * <p>
+     *
+     *
```

Comment:
Please replace by a single empty line. The same is true for the other comments in this class.